### PR TITLE
organize common code

### PR DESCRIPTION
**Pull request by giuliovn**

### CODE REVIEW

File: .github/workflows/workflows_validator.yml [hunk 1]
```diff
@@ -20,12 +20,12 @@ jobs:
       run: pip install boto3
 
     - name: Validate input templates
-      run: python .github/utils/validate_input_templates.py
+      run: python scripts/healthomics_wf/utils/validate_input_templates.py
```
Comment (avigailmo) [inline .github/workflows/workflows_validator.yml:23]: after this pr is merged need to update also the validation in terra_pipeline publish:
https://github.com/Ultimagen/terra_pipeline/blob/master/.github/workflows/terra_pipeline-release.yml#L313

File: .github/workflows/workflows_validator.yml [hunk 2]
```diff
@@ -20,12 +20,12 @@ jobs:
...
 
   validate-docker-images:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
 
     - name: Validate docker images
-      run: python .github/utils/validate_globals_docker_images.py workflows
+      run: python scripts/healthomics_wf/utils/validate_globals_docker_images.py workflows
```
Comment (avigailmo) [inline .github/workflows/workflows_validator.yml:31]: same

File: scripts/healthomics_wf/utils/validate_globals_docker_images.py [hunk 3]
```diff
@@ -3,39 +3,45 @@
...
-    logging.debug("Make sure global wdls are all the same")
+    logger.debug("Make sure global wdls are all the same")
     all_globals_wdls = glob(str(workflows_dir / "*" / GLOBALS_REL_PATH))
     assert len(set([file_md5(global_file) for global_file in all_globals_wdls])) == 1, "Not all globals file are the same"
-    logging.debug("Get docker images from globals.wdl")
-    docker_images = parse_wdl_for_images(all_globals_wdls[0])
-    logging.debug(f"Images found: {docker_images}")
+    logger.debug("Get docker images from globals.wdl")
+    docker_images_dict, _ = parse_wdl_for_images(all_globals_wdls[0])
+    docker_images = set(docker_images_dict.values())
```
Comment (avigailmo) [inline scripts/healthomics_wf/utils/validate_globals_docker_images.py:43]: 👍

File: scripts/healthomics_wf/utils/validate_globals_docker_images.py [hunk 4]
```diff
@@ -45,15 +51,6 @@ def file_md5(file_path: str) -> str:
     return file_md5
 
 
-def parse_wdl_for_images(wdl_file: Path) -> set:
```
Comment (avigailmo) [inline scripts/healthomics_wf/utils/validate_globals_docker_images.py:48]: why not to use the `parse_wdl_for_s3_paths` from `localize_s3.py` in the same way?

Comment (giuliovn) [reply scripts/healthomics_wf/utils/validate_globals_docker_images.py:48]: not sure I understand what you mean

**Pull request merged**